core/src/epicli/data/common/ansible/playbooks/roles/docker/defaults/main.yml

```
@@ -4,4 +4,8 @@ docker_logging:
     max_file_size: 10m # The maximum size of the log before it is rolled. A positive integer plus a modifier representing the unit of measure (k, m, or g)
     max_files: 2       # The maximum number of log files that can be present
 
-docker_version: 18.09.6
\ No newline at end of file
+docker_version: "5:18.09.*"
+docker_cli_version: "5:18.09.*"
+
+docker_version_rh: "3:18.09.*"
+docker_cli_version_rh: "1:19.03.*"
```

```suggestion
docker_version:
  Debian: "5:18.09.*"
  RedHat: "18.09.*"
```
We would like to have the same version for `docker-ce` and `docker-ce-cli`.